Refine OpenGov doc (#5513)
w3f/polkadot-wiki: docs/learn/learn-polkadot-opengov.md

Once the proposal exits the Lead-in Period and enters the Voting Period, to be approved, it must
satisfy the approval and support criteria for the **Confirmation Period**.

- **Approval** is defined as the share of [conviction](#voluntary-locking)-weighted _aye_ votes
  against the conviction-weighted total of _aye_ and _nay_ votes. The code implementation can be viewed
  [here](https://github.com/paritytech/polkadot-sdk/blob/f2fbba3be1d7deaf7cfc731cea00552c212ddfcf/substrate/frame/conviction-voting/src/types.rs#L77)
- **Support** is the total number of _aye_ and _abstain_ votes (ignoring any adjustment for
  conviction) compared to the total possible votes ([active issuance](learn-DOT.md#token-issuance))
  that could be made in the system. In case of _split_ votes, only _aye_ and _abstain_ will count.

For example, let us consider a hypothetical example where the total active issuance is {{ polkadot: 100 DOT :polkadot }}{{ kusama: 100 KSM :kusama }}

> and an account A with 10 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} votes "Aye" with 4x conviction, where as account B votes "Nay" with 5 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} with 2x conviction 

Reviewer: ```suggestion
and an account A with 10 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} votes "Aye" with 4x conviction, whereas account B votes "Nay" with 5 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} with 2x conviction 
```

Reviewer 2: This is "whereas". I'll edit it.

> and finally account C with 20 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} votes "abstain". In this scenario, only 45 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} from the total active issuance participated in voting on the

Reviewer: ```suggestion
and finally account C with 20 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} votes "abstain". In this scenario, only 35 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} from the total active issuance participated in voting on the
```

> referendum. Now, let us calculate the Approval and Support values for that referendum. Approval is calculated as Aye'/ Aye' + Nay', where Aye' and Nay' are the votes after 

Reviewer: ```suggestion
referendum. Now, let us calculate the Approval and Support values for that referendum. Approval is calculated as (Aye') / (Aye' + Nay')  where "Aye'" and "Nay'" are the votes after 
```

Reviewer 2: Aye' is Aye with conviction. It is important to differentiate this variable from just Aye which is in support formula, which corresponds to DOT used to vote Aye. I see you added in the description, but for someone who might only be looking at the formuals would not be aware that there is a difference between these variables.

> the conviction multiplier is applied. Hence, Approval = (10 x 4) / (10 x 4 + 5 x 2) = 40/50 which is 80%. Support is calculated as (Aye + Abstain) / (total active issuance). Hence,  Support = (10 + 20) / 100 which is 30%.

Reviewer: ```suggestion
the conviction multiplier is applied. Hence, Approval = (10 x 4) / (10 x 4 + 5 x 2) = 40/50 which is 80%. Support is calculated as (Aye + Abstain) / (total active issuance), where "Aye" and "Abstain" are the votes without the conviction multiplier. Hence,  Support = (10 + 20) / 100 which is 30%.
```
